src/controllers/certificate.controller.js

```diff
@@ -33,9 +34,10 @@ exports.generateGithubCert = async (req, res) => {
         if (repo.source) {
             throw new Error("Certificate can't be generated for forks");
         }
-        const [commits, pullRequests] = await Promise.all([
+        const [commits, pullRequests, lastCommit] = await Promise.all([

```

Comment:
You don't need  to call an extra API just for that. 

You can just extract the Latest commit form commits and latest PR from pullRequests then take the max of that.

Reply:
I thought of going with this idea first but in commits, there is no timestamp associated, therefore switched to the second method.

Reply:
Changed @Shubham-Kumar-2000